HiWARPs/cactus-backend
# Created endpoint to upload a CSV file.

# What does this PR do? 

Add endpoint to upload a CSV file. The content is then stored in mongo in 2 formats:
1 - "raw" : We take the file, convert it to JSON and store that.
2 - "dataPoints": We parse the file according to the content we expect, turn that to JSON for storage. 

The raw data is stored so that it is easy to debug if we have any parsing issues in the future.

# How to test this PR:

1 - start mongodb
2 - start this service
3 - send a file: Here is a sample CURL

     curl -i --location --request POST 'localhost:3000/file' --form 'file=@"sample.csv"'

4. Check that the data was stored.

# Anything else the reviewer should know about?

Yes! 

1 - I couldn't come up with a better name that "electrons" for the collection name.  If you know a better one, I can modify this PR for it. 

2 - This PR ONLY stores the data. It does not related it to a project or a form. That would go in a different PR.

3 - The headers in the CSV matters. The one I am using in my samples are

     X,YPi,YerrPi
